DfE.FindInformationAcademiesTrusts/DfE.FindInformationAcademiesTrusts.csproj

```diff
@@ -16,19 +16,23 @@
     </PropertyGroup>
 
     <ItemGroup>
-        <PackageReference Include="Microsoft.ApplicationInsights.AspNetCore" Version="2.21.0" />
-        <PackageReference Include="Serilog.AspNetCore" Version="7.0.0" />
-        <PackageReference Include="Serilog.Sinks.ApplicationInsights" Version="4.0.0" />
+        <PackageReference Include="Microsoft.ApplicationInsights.AspNetCore" Version="2.21.0"/>
+        <PackageReference Include="Serilog.AspNetCore" Version="7.0.0"/>
+        <PackageReference Include="Serilog.Sinks.ApplicationInsights" Version="4.0.0"/>
     </ItemGroup>
 
     <ItemGroup>
-      <Content Remove="package.json" />
-      <None Include="package.json">
-        <CopyToOutputDirectory>Never</CopyToOutputDirectory>
-      </None>
-      <Content Remove="package-lock.json" />
-      <None Include="package-lock.json">
-        <CopyToOutputDirectory>Never</CopyToOutputDirectory>
-      </None>
+        <Content Remove="package.json"/>
+        <None Include="package.json">
+            <CopyToOutputDirectory>Never</CopyToOutputDirectory>
+        </None>
+        <Content Remove="package-lock.json"/>
+        <None Include="package-lock.json">
+            <CopyToOutputDirectory>Never</CopyToOutputDirectory>
+        </None>
+        <Content Update="webpack.assets.config.js">
```

Comment:
This was added by the IDE, when adding webpack.assets.config.js and not sure if we want this

Reply:
We'll want it to be the same as the `package.json` - I'll show you how to do this in the IDE

Reply:
Should we do the same for webpack.config.js as well?